### PR TITLE
DBZ-1227 Upgrade to Kafka 2.2.0

### PR DESCRIPTION
**Pull request by Naros**
https://issues.jboss.org/browse/DBZ-1227

### CODE REVIEW

File: kafka/0.10/Dockerfile [hunk 1]
```diff
@@ -6,13 +6,13 @@ MAINTAINER Debezium Community
 # Set the version, home directory, and MD5 hash.
 # MD5 hash taken from http://kafka.apache.org/downloads.html for this version of Kafka
 #
-ENV KAFKA_VERSION=2.1.1 \
+ENV KAFKA_VERSION=2.2.0 \
     SCALA_VERSION=2.12 \
     CONFLUENT_VERSION=5.1.2 \
```
Comment (gunnarmorling) [inline kafka/0.10/Dockerfile]: Could you check whether there's an accompanying update of Confluent Platform? This is used for pulling in matching versions of the Avro serializers.

Comment (Naros) [reply kafka/0.10/Dockerfile]: @gunnarmorling Jiri and I did check and there is a deserialization issue by upgrading the platform at the moment, see https://issues.jboss.org/browse/DBZ-1237

Comment (gunnarmorling) [reply kafka/0.10/Dockerfile]: Ok, thanks!

**Comment by gunnarmorling** [comment]
Applied; thanks, @Naros!

**Pull request merged**